python/python/2_Functions_Classes_Modules.ipynb

```diff
@@ -428,6 +469,93 @@
     "pow2(5)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "subslide"
+    }
+   },
+   "source": [
+    "## Python typing\n",
+    "\n",
+    "since python 3.5 developers can provide [type hints](https://docs.python.org/3/library/typing.html) (float, int, tuple...). This is **optional**, unused at runtime.\n",
```

Comment:
```suggestion
    "Since Python 3.5 developers can provide [type hints](https://docs.python.org/3/library/typing.html) (float, int, tuple...). This is **optional**, unused at runtime.\n",
```

Not sure "unused at runtime" means something for beginners :shrug: